src/App.tsx

```diff
@@ -1,7 +1,36 @@
 import React from 'react';
 
-export const App: React.FC = () => (
-  <div className="App">
-    <p className="App__message">The last pressed key is [Enter]</p>
-  </div>
-);
+type State = {
+  pressedKey: string,
+};
+export class App extends React.Component<{}, State> {
+  state: State = {
+    pressedKey: '',
+  };
+
+  private NO_KEY_PRESSED = 'Nothing was pressed yet';
```

Comment:
I don't think you need to create variable for this message since you use it only once